src/hooks/useProtocolMetrics.ts

```diff
@@ -132,6 +132,18 @@ export const useTreasuryLiquidBackingPerOhmFloating = () =>
   useProtocolMetrics(metrics => metrics[0].treasuryLiquidBackingPerOhmFloating);
 export const useOhmCirculatingSupply = () => useProtocolMetrics(metrics => metrics[0].ohmCirculatingSupply);
 export const useOhmFloatingSupply = () => useProtocolMetrics(metrics => metrics[0].ohmFloatingSupply);
+
+/**
+ * @deprecated
+ * Please avoid using this method.
+ * you should be using `useOhmPrice` from `src/hooks/usePrices.ts`... as subgraph has an indexing delay
+ */
 export const useOhmPrice = () => useProtocolMetrics(metrics => metrics[0].ohmPrice);
+
+/**
+ * @deprecated
+ * Please avoid using this method.
+ * you should be using `useGohmPrice` from `src/hooks/usePrices.ts`... as subgraph has an indexing delay
+ */
 export const useGOhmPrice = () => useProtocolMetrics(metrics => metrics[0].gOhmPrice);
```

Comment:
didn't even realize these existed. I must have glossed over these since they're identically named to our other hook. ty for deprecating.

Reply:
yea i didn't either... my fault